scripts/app.js

```diff
@@ -34,3 +39,19 @@ projectObjects.forEach(function(projectsObj) {
 projects.forEach(function(projectsObj) {
   $('#projects').append(projectsObj.toHtml());
 });
+
+Article.fetchAll = function() {
```

Comment:
The fetchAll function as a method of a nonexistent Articles object so when it’s called in portfolio_view, it doesn’t do anything. Also, the loadAll and renderIndexPage functions haven’t been written yet.